### ISSUE TITLE
too early turn @ some waypoints = offtrack

### ISSUE THREAD

**Issue by andrejetcat** [flybywiresim/aircraft#6283]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-11-26T19:06:28+00:00",
  "ref": "master",
  "sha": "6803283d85f97decbccf60eb3162d701ec71b217",
  "actor": "sidnov",
  "event_name": "manual"
}
```

### Describe the bug

on todays flight, i see that the plane turns to early on the OGBER25R ARR @ EDDB. you can see it on the picture, that the plane starts turning too early and therefore its far off track. After some seconds and off-distance it try to turn back on track.
It happend also on some waypoints on track/DEP´s etc
![turn too early](https://user-images.githubusercontent.com/72918017/144034672-7bbf75be-0faf-458a-9afe-009848f4619a.jpg)

### Expected behavior

should turn no to early to stay on track

### Steps to reproduce

1. stary C&D
2. fly any route
3. look  if plane stays on route

**Comment by Benjozork** [comment]
This is a known issue with the geometry having transient states. We will fix this soon with a coming LNAV update.

**Comment by derl30n** [comment]
Has this issue been solved?

**Comment by 2hwk** [comment]
Should be solved as of cFMS 1.5, definitely by fms v2.
